UnitTests/Drawing/CellTests.cs

```diff
@@ -201,43 +204,44 @@ public void RuneCell_LoadRuneCells_Without_ColorScheme_Is_Never_Null ()
 
         for (var i = 0; i < tv.Lines; i++)
         {
-            List<RuneCell> line = tv.GetLine (i);
+            List<Cell> line = tv.GetLine (i);
 
-            foreach (RuneCell rc in line)
+            foreach (Cell c in line)
             {
-                Assert.NotNull (rc.ColorScheme);
+                Assert.NotNull (c.Attribute);
             }
         }
     }
 
     [Fact]
     [AutoInitShutdown]
-    public void RuneCellEventArgs_WordWrap_True ()
+    public void CellEventArgs_WordWrap_True ()
```

Comment:
Is it possible to refactor this test to not use `TextView`. IMO, unit tests for a class hsould not depend on any other classes.

If this is really a TV test, then move it to TV tests.